core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -226,6 +228,11 @@ var StoryEditorPage = function() {
     explorationIdSaveButton.click();
   };
 
+  this.changeNodeDescription = function(nodeDescription) {
+    nodeDescriptionInputField.clear();
```

Comment:
Can you add a `waitFor` here to make sure the field is visible and editable before you try and change it? Missing these `watiFor` lines is a cause of a lot of flakiness

Reply:
Done.